user/conditions-v0.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Conditions (v0, deprecated)
+layout: en
+---
+
+> This page documents the behavior of conditions `v0` (beta), which is
+> currently being deprecated. Please opt in to the new version `v1` using
+> `conditions: v1` in your `.travis.yml` file. The documentation for `v1`
+> can be found [here](/user/conditions-v1).
+
+Conditions can be used to filter out, and reject builds, stages, and jobs by
+specifying conditions in your build configuration (your `.travis.yml` file).
+See [Conditional Builds, Stages, and Jobs](/user/conditional-builds-stages-jobs)
+for details.
+
+## Specifying conditions
+
+The condition can be specified using a boolean language as follows:
+
+```
+(NOT [term] OR [term]) AND [term]
+```
+
+A term is defined as:
+
+```
+[left-hand-side] [operator] [right-hand-side]
+```
+
+All keywords (such as `AND`, `OR`, `NOT`, `IN`, `IS`, attributes, and
+functions) are case-insensitive.
+
+### Left hand side
+
+The left hand side part can either be a known attribute or a function call.
+
+Known attributes are:
+
+* `type` (the current event type, known event types are: `push`, `pull_request`, `api`, `cron`)
+* `repo` (the current repository slug `owner_name/name`)
+* `branch` (the current branch name; for pull requests: the base branch name)
+* `tag` (the current tag name)
+* `sender` (the event sender's login name)
+* `fork` (`true` or `false` depending if the repository is a fork)
+* `head_repo` (for pull requests: the head repository slug `owner_name/name`)
+* `head_branch` (for pull requests: the head repository branch name)
+
+Known functions are:
+
+* `env(FOO)` (the value of the environment variable `FOO`)
+
+The function `env` currently only supports environment variables that are given
+in your build configuration (e.g. on `env` or `env.global`), not environment
+variables specified in your repository settings.
+
+### Right hand side
+
+It is currently not possible to compare function calls. This means that if you
+try to evaluate something similar to:
+
+`env(PRIOR_VERSION) != env(RELEASE_VERSION)`
+
+where `PRIOR_VERSION` and `RELEASE_VERSION` are environment variables defined
+elsewhere in .travis.yml, the condition will be evaluated as true, even when it
+is false.
+
+A work-around is to create a script which expresses the conditional test in
+reverse, and applies `travis_terminate` when the condition is false. This
```

Comment:
hm, can we clarify/define the purpose of `travis_terminate` before this example/workaround? Otherwise, this is  🆒

Reply:
see above :)

Reply:
right! so, re: `travis_terminate` - I wanted to be ultra clear on what it does. From what I can see, `travis_terminate` isn't documented anywhere in the `Deploys` docs and it only turns up for on the Conditionals page in `docs`, and in the code in `travis-build`(I searched via codesearch).

Given that we're referencing it in this example, though, I think we should clarify this: 
> and applies `travis_terminate` when the condition is false.
to:
> and applies `travis_terminate` when the condition is false. (`travis_terminate` is a key word which fails the build)

Reply:
As we discussed, this can be it's own topic: https://github.com/travis-ci/docs-travis-ci-com/issues/1899